app/views/govuk_publishing_components/components/docs/radio.yml

```diff
@@ -134,6 +134,19 @@ examples:
         text: "Yes"
       - value: "no"
         text: "No"
+  with_custom_page_header_sizes:
+    data:
```

Comment:
I think having the `is_page_heading` option here detracts from the point of the example. Maybe add a description to say something like `If heading_size is passed, size is set to that value. If is_page_heading is passed instead, the size defaults to xl`.